### PR TITLE
Resource page to access template

### PR DESCRIPTION
**Pull request by anissa-agahchen**
# Overview

## Links to jira tickets

- basic resource page with the 6 templates in a table accessing s3.
- still pending the design of the resources page

### CODE REVIEW

File: app/src/features/permits/PermitsPage.test.tsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ const mockBiohubApi = ((useBiohubApi as unknown) as jest.Mock<typeof mockUseBioh
   mockUseBiohubApi
 );
 
-describe('ProjectsListPage', () => {
+describe('PermitsPage', () => {
```
Comment (sdevalapurkar) [inline app/src/features/permits/PermitsPage.test.tsx:21]: nice catch!

File: app/src/features/resources/ResourcesLayout.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import Box from '@material-ui/core/Box';
```
Comment (sdevalapurkar) [inline app/src/features/resources/ResourcesLayout.tsx:1]: not an issue or question pertaining specifically to this pr, but just curious about if we can just have a single layout component cause all the layout components (resources, permits, projects, etc) are all the same code pretty much right?

File: app/src/features/resources/ResourcesPage.tsx [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+              <TableCell>Id</TableCell>
+              <TableCell>Name</TableCell>
+              <TableCell>Link</TableCell>
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            <TableRow>
+              <TableCell colSpan={6}>
+                <Box display="flex" justifyContent="center">
+                  No Results
```
Comment (sdevalapurkar) [inline app/src/features/resources/ResourcesPage.tsx]: should this maybe be:
```suggestion
                  No Resources
```
or
```suggestion
                  No Resource Results
```

File: app/src/features/resources/ResourcesPage.tsx [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+              ))}
+            </TableBody>
+          </Table>
+        </TableContainer>
+      );
+    }
+  };
+
+  /**
+   * Displays permits list.
```
Comment (sdevalapurkar) [inline app/src/features/resources/ResourcesPage.tsx]:
```suggestion
   * Displays resources list.
```

File: app/src/features/resources/ResourcesRouter.tsx [hunk 5]
```diff
@@ -0,0 +1,35 @@
+//import CreatePermitPage from 'features/permits/CreatePermitPage';
```
Comment (sdevalapurkar) [inline app/src/features/resources/ResourcesRouter.tsx]:
```suggestion
```

File: app/src/features/resources/ResourcesRouter.tsx [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+import AppRoute from 'utils/AppRoute';
+import PrivateRoute from 'utils/PrivateRoute';
+import ResourcesPage from './ResourcesPage';
+
+interface IResourcesRouterProps {
+  classes: any;
+}
+
+/**
+ * Router for all `/resources/*` pages.
```
Comment (sdevalapurkar) [inline app/src/features/resources/ResourcesRouter.tsx]:
```suggestion
 * Router for all `/admin/resources/*` pages.
```

File: app/src/features/resources/ResourcesLayout.tsx (hunk 2 above)
Comment (anissa-agahchen) [reply app/src/features/resources/ResourcesLayout.tsx:1]: yeah .. at the moment, we don't have a layout for the resources yet.  But good idea.

**Review by NickPhura — APPROVED** [review]

**Pull request merged**